src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -191,7 +192,7 @@ private void ReportRequiresDiagnostic (OperationAnalysisContext operationContext
 		private string GetUrlFromAttribute (AttributeData? requiresAssemblyFilesAttribute)
 		{
 			var url = requiresAssemblyFilesAttribute?.NamedArguments.FirstOrDefault (na => na.Key == "Url").Value.Value?.ToString ();
```

Comment:
I guess here there are two things wrong first the parameter of the method is called requiresAssemblyFilesAttribute instead of requiresAttribute which is probably my fault, and second the function is called GetRucUrlArg (and there is a GetRucMessageArg). GetUrlFromAttribute is in the base because is generic to whichever RequiresAttribute so calling a method with Ruc on the name sounds funky maybe we should call them GetRequiresUrlArg/GetRequiresMessageArg